release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -5962,3 +5962,102 @@ openshift3/ose-template-service-broker:v3.11.705-1.g7a17a5d
 ==== Upgrading
 
 To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest release, see xref:../upgrading/index.adoc#install-config-upgrading-index[Upgrade methods and strategies] for instructions.
+
+[[ocp-3-11-715]]
+=== RHBA-2022:5000 - {product-title} 3.11.715 bug fix and security update
+
+Issued: 2022-06-20
+
+{product-title} release 3.11.715 is now available. The list of packages and bug fixes included in the update are documented in the link:https://access.redhat.com/errata/RHBA-2022:5000[RHBA-2022:5000] advisory. The container images included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:4999[RHSA-2022:4999] advisory.
+
+[[ocp-3-11-715-bug-fix]]
```

Comment:
Normally if you were to have a bug fix it would go under the image list, but this is a security response (which is why there is no "fix" you can actually remove this.

Reply:
Addressed